contracts/ThreeHeapOrdering.sol

```diff
@@ -163,12 +163,9 @@ library ThreeHeapOrdering {
         uint256 size = _heap.size;
         uint256 accountsLength = _heap.accounts.length;
```

Comment:
Does it cost more to remove this var?

Reply:
Yes because we push just after and so we would replace it with `_heap.accounts.length -1`
![Uploading remlen.png…]()

Reply:
Hmm I'm not sure about the link you posted

Reply:
> Hmm I'm not sure about the link you posted

Yes sorry, upload did not go through somehow

Reply:
Ok wanted to be sure thx